torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4311,6 +4311,26 @@ def sample_inputs_narrow_copy(op_info, device, dtype, requires_grad, **kwargs):
         yield SampleInput(tensor, args=args)
 
 
+def sample_inputs_view_copy(op_info, device, dtype, requires_grad, **kwargs):
```

Comment:
Can we reuse https://github.com/pytorch/pytorch/blob/fca6ed02b91e0d685cc9b8b504bac5d356d31876/torch/testing/_internal/common_methods_invocations.py#L6050?

Reply:
reused